src/System.Text.Utf8String/System/Text/Utf8String.cs

```diff
@@ -766,7 +804,7 @@ public Utf8String Trim(byte[] trimCodeUnits)
         // TODO: Name TBD, CopyArray? GetBytes?
         public byte[] CopyBytes()
         {
-            return _buffer.ToArray();
+            return _buffer;
```

Comment:
This does not copy. 

But as I commented above, I think this should be acomplished with utf8.Bytes.ToArray();